.github/workflows/deploy.yml

```
@@ -97,14 +98,26 @@ jobs:
             echo "platforms=linux/arm64" >> $GITHUB_OUTPUT
           fi
 
-      - name: Build & push image
+      - name: Build & push image to ECR
+        id: build-ecr
+        uses: docker/build-push-action@v4
```

actionlint (1.7.12)

[error] the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)